src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -1461,7 +1452,7 @@ public static void Main()
 }
 
 class Person { public string Name; }
-class Teacher : Person { public string Subject; }
+class Teacher : Person { public string Subject = string.Empty; }
```

Comment:
>= string.Empty; [](http://example.com/codeflow?start=47&length=15)

Is this test change needed?

Reply:
I think it's better to avoid reporting and validating unrelated warnings.